packages/realm/src/scripts/build/cli.ts

```diff
@@ -0,0 +1,185 @@
+////////////////////////////////////////////////////////////////////////////
```

Comment:
I think we can close https://github.com/realm/realm-js/issues/3708 😄

Reply:
I'd like to get the Node build into this CLI too, before closing #3708